src/FileHelper.php

```diff
@@ -412,16 +411,41 @@ public static function copyFile(string $source, string $destination, array $opti
                 chmod($destination, $fileMode);
             }
 
-            if ($afterCopy) {
-                $afterCopy($source, $destination);
-            }
+            self::processCallback($afterCopy, $source, $destination);
 
             return true;
         }
 
         return false;
     }
 
+    /**
+     * @param mixed $callback
+     * @param array $arguments
+     *
+     * @throws InvalidArgumentException
+     *
+     * @return mixed
+     */
+    private static function processCallback($callback, ...$arguments)
+    {
+        if ($callback === null) {
+            return;
+        }
+
+        if ($callback instanceof Closure) {
+            return $callback(...$arguments);
+        }
+
+        if (is_callable($callback)) {
+            return call_user_func_array($callback, $arguments);
+        }
+
+        $type = \is_object($callback) ? \get_class($callback) : \gettype($callback);
```

Comment:
```suggestion
        $type = \is_object($callback) ? \get_class($callback) : \gettype($callback);
```

Use `use` section for functions instead FQN.

Reply:
> Use `use` section for functions instead FQN.

Strange. "Fix uses" don't auto-resolve it